packages/angular-sdk-components/src/lib/_components/widget/file-utility/file-utility.component.ts

```diff
@@ -702,9 +708,11 @@ export class FileUtilityComponent implements OnInit, OnDestroy {
     return this.pConn$.getValue(PCore.getConstants().CASE_INFO.CASE_INFO_ID) || '';
   }
 
-  fetchCaseAttachments(caseID) {
+  async fetchCaseAttachments(caseID) {
     const attachmentUtils = PCore.getAttachmentUtils();
-    return attachmentUtils.getCaseAttachments(caseID, this.pConn$.getContextName());
+    // eslint-disable-next-line sonarjs/prefer-immediate-return
+    const attachments = await attachmentUtils.getCaseAttachments(caseID, this.pConn$.getContextName());
+    return attachments;
```

Comment:
I can't see any difference between the old code and the new code. They both do the same thing.